### PR TITLE
Add Matrix import to the existing Gitter APi import

### PR DESCRIPTION
**Pull request by timbl**
belated creation of matrix branch

### CODE REVIEW

**Comment by timbl** [comment]
see https://github.com/solid-contrib/gitter-solid/issues/34

WIP

File: gitter-solid.js [hunk 1]
```diff
@@ -389,7 +367,8 @@ async function handleMatrixMessage (event, room, config) {
         return
     }
 
-    const relatesTo = content['m.relates_to']
+    // Found a case where a thread rel was just misssung fomr the getContrnt() version
```
Comment (TallTed) [inline gitter-solid.js]:
```suggestion
    // Found a case where a thread rel was just missing from the `getContrnt()` version
```

File: gitter-solid.js [hunk 2]
```diff
@@ -1645,6 +1692,9 @@ async function go () {
 } // go
 
 var toBePut = []
+var toBeLinked = [] // List messages to be linked to thngs we only have matrix ID for
+var toBeExecuted = [] //functions to be run  when we know eventMap
+var eventMap = {} // Map eventid to solid URI
```
Comment (TallTed) [inline gitter-solid.js]:
```suggestion
var toBeLinked = [] // List messages to be linked to things we only have matrix ID for
var toBeExecuted = [] // functions to be run when we know `eventMap`
var eventMap = {} // Map `eventid` to solid URI
```

**Comment by timbl** [comment]
Basically working, not perfect. Mights as well push to main from now on

**Comment by TallTed** [comment]
It's unclear whether "Mights as well push to main from now on" means you will be pushing immediately, or going through PRs.  I strongly recommend the latter, as this is not a single-coder nor even a single-committer project, and PRs allow for such easy and fast corrections as remain frustratingly unapplied, above.

Typos, especially in code, even in comments therein, impact later coders, and should be fixed.

**Pull request merged**